src/components/FollowBar/FollowBar.jsx

```diff
@@ -1,51 +1,42 @@
 import styles from "./FollowBar.module.css";
 import Avatar from "@mui/material/Avatar";
 import Button from "@mui/material/Button";
+import { useDispatch, useSelector } from "react-redux";
+import { followUser, getAllUsers } from "firebaseMethods";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+
 export const FollowBar = () => {
-  const people = [
-    {
-      id: 1,
-      name: "Sadath",
-    },
-    {
-      id: 2,
-      name: "Nisha",
-    },
-    {
-      id: 3,
-      name: "Aish",
-    },
-    {
-      id: 4,
-      name: "Jenny",
-    },
-    {
-      id: 5,
-      name: "Jyo",
-    },
-    {
-      id: 6,
-      name: "Chanchal",
-    },
-    {
-      id: 7,
-      name: "Mazz",
-    },
-    {
-      id: 8,
-      name: "Yash",
-    },
-  ];
+  const dispatch = useDispatch();
+  const { allusers } = useSelector((store) => store.allUsers);
+  const { user, token } = useSelector((store) => store.user);
+
+  const findUserDoc = allusers.find((u) => u?.id === token);
+
+  const filterUser = allusers.filter((u) => u?.id !== token);
+
+  const people = filterUser.filter(
+    (person) => !user?.following?.some((p) => p.userId === person.id)
+  );
+
+  const trimPeople = people.length > 4 ? people.splice(0, 4) : people;
+
+  const handleFollow = (peer) => {
+    followUser(user, findUserDoc.id, peer?.id, peer?.data, dispatch);
+  };
+
   return (
     <>
       <div className={`${styles.followbar_container}`}>
         <h3>Suggestions</h3>
         <ul className="list">
-          {people?.map(({ id, name }) => (
-            <li key={id} className={`${styles.user_info}`}>
-              <Avatar alt={name} />
-              <p>{name}</p>
-              <Button variant="contained" >
+          {trimPeople?.map((peer) => (
+            <li key={peer.id} className={`${styles.user_info}`}>
+              <Avatar alt={peer?.data?.userName} />
```

Comment:
Are you missing the src attribute here?

Reply:
Yeah left it intentionally, working on image part